public/pages/CreateTrigger/containers/DefineTrigger/utils/validation.js

```diff
@@ -4,27 +4,22 @@
  */
 import _ from 'lodash';
 
-import { FORMIK_INITIAL_TRIGGER_VALUES, TRIGGER_TYPE } from '../../CreateTrigger/utils/constants';
-
-export const validateTriggerName = (triggers, triggerToEdit, fieldPath, isFullText) => (value) => {
```

Comment:
I believe we do still want to have an `isFullText` parameter for this validation function. It's not used by most calls to `validateTriggerName`, but we can see that we do provide that parameter [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blame/main/public/pages/CreateTrigger/containers/DefineTrigger/DefineTrigger.js#L242) when defining a trigger in `flyoutMode`.

Looks like that parameter was added in commit https://github.com/opensearch-project/alerting-dashboards-plugin/commit/12c65ecf4ab25353874e616f37c1feafd33925bc to provide a validation error that takes up less space when defining a trigger in a flyout window.

Reply:
can we have a case that how this would be used with? i try to console the `flyoutMode` and see how it interact with the validation, but i didnt get it.

Reply:
@lezzago correct me if I'm wrong, but I think this is just used by "feature anywhere" at the moment, right? 

If so, this parameter wouldn't be used by the alerting UI pages directly, as I don't believe we have any places where we configure triggers in flyouts. "Feature anywhere" allows other pages of the dashboards site to use alerting UI assets so users can define monitors and triggers directly from graphs and other visualizations.